livechat/reports/client.py

```diff
@@ -0,0 +1,457 @@
+                    groups: str = None,
+                    customer_client_ids: str = None,
+                    tags: str = None,
+                    tagged: str = None,
+                    payload: dict = None,
+                    headers: dict = None) -> httpx.Response:
+        ''' Shows how many chats occurred during the specified period.
+
+        Args:
+            to (str): Date in the RFC3339 format, which also contains a timezone.
+                      This timezone will be used if no `timezone` is provided.
+            from_param (str): Date in the RFC3339 format, which also contains a timezone.
+                              This timezone will be used if no `timezone` is provided.
+            distribution (str): Possible values: `hour`, `day-hours`, `day`, `month`; defaults to `day`.
+            timezone (str): Timezone in the TZ format (e.g. America/Phoenix).
+                            Defaults to the requester's timezone. If not present, `from` is parsed to get the requester's timezone.
+            agents (str): Agent emails separated by a comma.
+                          If not specified, returns the data for all agents within the license.
+            agents_assigned (str): Possible values: `true`, `1`, `false`, `0`
+            groups (str): Group IDs separated by a comma.
+            customer_client_ids (str): Client IDs separated by a comma.
+            tags (str): Names of tags separated by a comma.
+            tagged (bool): Possible values: `true`, `1`, `false`, `0`
+            payload (dict): Custom payload to be used as request's data.
+                            It overrides all other parameters provided for the method.
+            headers (dict): Custom headers to be used with session headers.
+                            They will be merged with session-level values that are set,
+                            however, these method-level parameters will not be persisted across requests.
+
+        Returns:
+            httpx.Response: The Response object from `httpx` library,
+                            which contains a server’s response to an HTTP request.
+        '''
+        if payload is None:
+            payload = prepare_payload(locals())
+            if from_param is not None:
+                del payload['from_param']
+                payload['from'] = from_param
+        else:
+            if 'from_param' in payload:
+                payload['from'] = payload['from_param']
+                del payload['from_param']
+        return self.session.get(f'{self.api_url}/chats/total_chats',
+                                params=payload,
+                                headers=headers)
+
+
+class ReportsAPI34(ReportsApiInterface):
+    ''' Configuration API client in version 3.4 class. '''
+
+    # Chats
+
+    def duration(self,
+                 distribution: str = None,
+                 timezone: str = None,
+                 filteters: dict = None,
```

Comment:
```suggestion
                 filters: dict = None,
```
Please fix all typos in the module.